### PR TITLE
20241015 dependency updates

### PR DESCRIPTION
**Pull request by michaelwebb76**
This failed locally for me with `zlib` dependency problems, but perhaps it will work in CI.

### CODE REVIEW

File: servant-activeresource.cabal [hunk 1]
```diff
@@ -47,7 +47,7 @@ common deps
     , containers        ^>=0.6     || ^>=0.7
     , servant           >=0.19     && <0.21
     , servant-server    >=0.19     && <0.21
-    , template-haskell  >=2.16     && <2.22
+    , template-haskell  >=2.16     && <2.23
```
Comment (JackKelly-Bellroy) [inline servant-activeresource.cabal:50]: `template-haskell` is coupled to GHC versions so I don't think we should raise this until we support 9.10.

**Review by JackKelly-Bellroy — COMMENTED** [review]
Weird, the shell worked on my machine. Once the `template-haskell` bump goes away, there won't be much in this PR.

File: servant-activeresource.cabal (hunk 1 above)
Comment (michaelwebb76) [reply servant-activeresource.cabal:50]: K thanks

Comment (michaelwebb76) [reply servant-activeresource.cabal:50]: How do you know which packages are tied to versions of GHC?

**Comment by michaelwebb76** [comment]
This only contains an update to `template-haskell` bounds which should be synchronized with an update to GHC

**Pull request closed without merging**